src/lib/components/MultiCodeContextless.svelte

```diff
@@ -0,0 +1,93 @@
+<script lang="ts">
+    import { platformMap } from '$lib/utils/references';
+    import { writable } from 'svelte/store';
+    import { getCodeHtml, type Language } from '$lib/utils/code';
+    import { copy } from '$lib/utils/copy';
+    import { Select, Tooltip } from '$lib/components';
+
+    export let selected: Language = 'js';
+    export let data: { language: string; content: string; platform?: string }[] = [];
+    export let width: number | null = null;
+    export let height: number | null = null;
+
+    $: snippets = writable(new Set(data.map((d) => d.language)));
+
+    $: content = data.find((d) => d.language === selected)?.content ?? '';
+
+    $: platform = data.find((d) => d.language === selected)?.platform ?? '';
+
+    snippets?.subscribe((n) => {
+        if (selected === null && n.size > 0) {
+            selected = Array.from(n)[0] as Language;
+        }
+    });
+
+    enum CopyStatus {
+        Copy = 'Copy',
+        Copied = 'Copied!'
+    }
+    let copyText = CopyStatus.Copy;
+    async function handleCopy() {
+        await copy(content);
+
+        copyText = CopyStatus.Copied;
+        setTimeout(() => {
+            copyText = CopyStatus.Copy;
+        }, 1000);
+    }
```

Comment:
Same comment as in https://github.com/appwrite/website/pull/1485#discussion_r1848548939

Reply:
👍🏼